remote-method/index.js

```diff
@@ -0,0 +1,285 @@
+      var done = this.async();
+      var prompts = [
+        {
+          name: 'model',
+          message: 'Select the model:',
+          type: 'list',
+          choices: this.modelNames
+        }
+      ];
+      this.prompt(prompts, function(answers) {
+        this.modelName = answers.model;
+        done();
+      }.bind(this));
+    }
+  },
+
+  findModelDefinition: function() {
+    this.modelDefinition = this.projectModels.filter(function(m) {
+        return m.name === this.modelName;
+    }.bind(this))[0];
+
+    if (!this.modelDefinition) {
+        var msg = 'Model not found: ' + this.modelName;
+        this.log(chalk.red(msg));
+        this.async()(new Error(msg));
+    }
+  },
+
+  askForParameters: function() {
+    var done = this.async();
+    var prompts = [
+      {
+        name: 'methodName',
+        message: 'Enter the remote method name:',
+        required: true,
+        validate: function(value) {
+          return validateName(value);
+        }
+      },
+      {
+        name: 'isStatic',
+        message: 'Is Static?',
+        required:true,
+        type: 'confirm',
+        default: true
+      },
+      {
+        name: 'description',
+        message: 'Description for method:'
+      }
+    ];
+    this.prompt(prompts, function(answers) {
+      this.methodName = answers.methodName;
+      this.isStatic = answers.isStatic;
+      this.description = answers.description;
+      done();
+    }.bind(this));
+  },
+
+  delimEndpoints: function() {
+    this.log('Let\'s add some Endpoints now.\n');
+  },
+
+  askForEndpoints: function() {
+    var done = this.async();
+
+    var prompts = [
+      {
+        name: 'httpPath',
+        message: 'Enter path for endpoints',
+        required:true
+      }
+    ];
+    this.prompt(prompts, function(answers) {
+      if (answers.httpPath == null || answers.httpPath === '') {
+        return done();
+      }
+      var subprompts = [
+        {
+          name: 'httpVerb',
+          message: 'HTTP verb:',
+          type: 'list',
+          choices: ['post', 'put', 'delete', 'get', {
+            name: '(other)',
+            value: null
+          }]
+        },
+        {
+          name: 'customHttpVerb',
+          message:
+          'Enter the custom http verb',
+          validate: validateName,
+          when: function(answers) {
+            return answers.httpVerb === null;
+          }
+        }
+      ];
+
+      var httpPath = answers.httpPath;
+
+      this.prompt(subprompts, function(answers) {
+        https.push({
+          'httpPath': httpPath,
+          'httpVerb': answers.httpVerb || answers.customHttpVerb
+        });
+        console.log(https);
+        this.log('\nLet\'s add another endpoint.');
+        this.askForEndpoints();
+      }.bind(this));
+    }.bind(this));
+  },
+
+  delimAccepts: function() {
+    this.log('Let\'s add some Accept Arguments now.\n');
+  },
+
+  askForAccepts: function() {
+    var done = this.async();
```

Comment:
Add info for the user how to break the loop. See e.g. the model generator ([source](https://github.com/strongloop/generator-loopback/blob/99c66a66d4d1e1179749d792ac3c298fd825636c/model/index.js#L170)).

```js
var done = this.async();
this.log('Enter an empty name when done.');
```